python/pysrc/kaskada/sources/arrow.py

```diff
@@ -246,6 +246,10 @@ def __init__(
             strings_can_be_null=True,
         )
 
+    _parse_options = pyarrow.csv.ParseOptions(
+        escape_char="\\",
+    )
+
```

Comment:
this was done to allow escaping comma's in CSV to handle JSON.  should I remove for now since we aren't adding json anymore?

Reply:
seems OK to leave. fine either way.